src/qt/paymentserver.cpp

```diff
@@ -315,19 +315,22 @@ void PaymentServer::handleURIOrFile(const QString& s)
                     tr("Payment request fetch URL is invalid: %1").arg(fetchUrl.toString()),
                     CClientUIInterface::ICON_WARNING);
             }
-#else
-            Q_EMIT message(tr("URI handling"),
-                tr("Cannot process payment request because BIP70 support was not compiled in."),
-                CClientUIInterface::ICON_WARNING);
-#endif
             return;
         }
+#endif
```

Comment:
Just from reading this, seems this is invalid when `ENABLE_BIP70` is disabled, due to the unpaired `else` block. Maybe we should disable BIP70 in one of the CI runs?